src/blocks/gallery-masonry/styles/style.scss

```diff
@@ -1,44 +1,105 @@
-.wp-block-coblocks-gallery-masonry {
-	position: relative;
+// Import styles for rendering the static content of deprecated gallery versions.
+@import "../v1/deprecated.scss";
 
-	ul {
-		list-style: none !important;
-		padding: 0 !important;
+figure.wp-block-coblocks-gallery-masonry.masonry-grid {
+	column-fill: initial;
```

Comment:
I see a lot of code duplication in the editor.scss and style.scss files. Is the backend only loading the editor.scss and the frontend only loading the style.scss? I'm afraid we do a fix on the editor.scss and forget style.scss or something like that. How do we handle that usually?

Reply:
This was just oversight on my part most likely. Thank you for finding this. The editor.scss should load only in the editor whereas the style.scss should load in both the editor as well as the front end.